packages/typespec-azure-specs-rulesets/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@azure-tools/typespec-azure-specs-rulesets",
```

Comment:
What do people think of the name

Reply:
Why not typespec-azure-rulesets?

I am not strong on any recommendation here, both are plain in their meaning. Just seems like all rulesets apply to specs.

Reply:
yeah I like it, renamed